src/main/java/com/jarslab/maven/babel/plugin/BabelTranspiler.java

```diff
@@ -13,70 +14,48 @@
 import java.io.UncheckedIOException;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
-import java.nio.file.Path;
 
-import static java.util.Objects.requireNonNull;
+import static java.lang.String.format;
+
+@Builder
+class BabelTranspiler {
 
-class BabelTranspiler
-{
     private static final String JAVASCRIPT_MIME_TYPE = "text/javascript";
     private static final String INPUT_VARIABLE = "input";
     private static final String BABEL_EXECUTE = "Babel.transform(%s, {presets: [%s]}).code";
```

Comment:
I'm thinking we could probably quite easily read the `option` parameter of `Babel.transform` (which is currently only `{presents: [...]}`) from a user-configurable file, allowing much greater flexibility.

Reply:
Good idea, I was thinking about that originally but frankly have no needs to implement it. Probably case for another PR?

Reply:
Definately something for a separate PR